tools/rosbag/src/player.cpp

```diff
@@ -106,8 +106,11 @@ Player::Player(PlayerOptions const& options) :
     // If we were given a list of topics to pause on, then go into that mode
     // by default (it can be toggled later via 't' from the keyboard).
     pause_for_topics_(options_.pause_topics.size() > 0),
+    pause_change_requested_(false),
+    requested_pause_state_(false),
     terminal_modified_(false)
 {
+  pause_service_ = node_handle_.advertiseService("pause", &Player::pauseCallback, this);
```

Comment:
Obviously a different service name here.

Perhaps another option might be:

```
ros::NodeHandle private_node_handle('~');
pause_service_ = private_node_handle.advertiseService("pause", &Player::pauseCallback, this);
```

This should get you a `rosbag/pause` service name, but it will be trivial to rename/remap it from roslaunch or wherever else.